go.mod

```diff
@@ -214,4 +214,7 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-replace github.com/confluentinc/ccloud-sdk-go-v2/networking-access-point => github.com/confluentinc/ccloud-sdk-go-v2-internal/networking-access-point v0.0.1
+replace (
+	github.com/confluentinc/ccloud-sdk-go-v2/networking => github.com/confluentinc/ccloud-sdk-go-v2-internal/networking v0.0.34
+	github.com/confluentinc/ccloud-sdk-go-v2/networking-access-point => github.com/confluentinc/ccloud-sdk-go-v2-internal/networking-access-point v0.0.3
```

Comment:
Ready to make these public?

Reply:
cc: @syoung-cflt

Reply:
@ejagade When do we make the SDK public?

Reply:
once the API docs are available public. ETA - End of this week.